provider/testdata/Cargo.toml

```diff
@@ -299,65 +300,73 @@ displaydoc = { version = "0.2.3", default-features = false, optional = true }
-icu_collections = { version = "1.0.0-beta1", path = "../../components/collections", optional = true }
-litemap = { version = "0.5", path = "../../utils/litemap", optional = true, default-features = false }
-tinystr = { version = "0.6", path = "../../utils/tinystr", optional = true }
-zerovec = { version = "0.8", path = "../../utils/zerovec", optional = true }
+icu_locid = { version = "1.0.0-beta1", path = "../../components/locid" }
+icu_collections = { version = "1.0.0-beta1", path = "../../components/collections" }
+litemap = { version = "0.5", path = "../../utils/litemap", default-features = false }
+tinystr = { version = "0.6", path = "../../utils/tinystr" }
+zerovec = { version = "0.8", path = "../../utils/zerovec" }
 
-[dev-dependencies]
-icu_plurals = { version = "1.0.0-beta1", path = "../../components/plurals" }
+# databake options
+icu_calendar = { version = "1.0.0-beta1", path = "../../components/calendar", default-features = false, optional = true }
+icu_casemapping = { version = "0.2", path = "../../experimental/casemapping", default-features = false, optional = true }
+icu_collator = { version = "1.0.0-beta1", path = "../../components/collator", default-features = false, optional = true }
+icu_datetime = { version = "1.0.0-beta1", path = "../../components/datetime", features = ["experimental"], default-features = false, optional = true }
+icu_decimal = { version = "1.0.0-beta1", path = "../../components/decimal", default-features = false, optional = true }
+icu_list = { version = "1.0.0-beta1", path = "../../components/list", default-features = false, optional = true }
+icu_locid_transform = { version = "1.0.0-beta1", path = "../../components/locid_transform", default-features = false, optional = true }
+icu_normalizer = { version = "1.0.0-beta1", path = "../../components/normalizer", default-features = false, optional = true }
+icu_plurals = { version = "1.0.0-beta1", path = "../../components/plurals", default-features = false, optional = true }
+icu_properties = { version = "1.0.0-beta1", path = "../../components/properties", default-features = false, optional = true }
+icu_segmenter = { version = "1.0.0-alpha1", path = "../../experimental/segmenter", features = ["lstm"], default-features = false, optional = true }
+icu_timezone = { version = "1.0.0-beta1", path = "../../components/timezone", default-features = false, optional = true }
 
 [features]
-default = ["static"]
-# Enables programmatic access to this Cargo.toml file.
-# If you want this to be made public stable, file an issue on the ICU4X repository.
-metadata = [
-    "cargo_metadata",
-    "icu_locid/serde",
-    "serde_json",
-    "serde",
-    "displaydoc",
-    "writeable",
-    "std"
-]
-fs = ["icu_provider/deserialize_json", "icu_provider_fs", "std", "lazy_static"]
-static = [
-    "icu_provider_blob",
-    "icu_provider_adapters",
-    "icu_provider_adapters/serde",
-    "lazy_static",
-]
-baked = [
+default = [
```

Comment:
Discussion: I am unsure about whether `icu_testdata` should all these features enabled by default.

- Build time is harmed by having all these default features
- But developer experience is better, and this is one of the first things developers will try

Reply:
Our icu_testdata usage is non-standard, as every change we make to a component requires a recompilation, so I'm seeing these features as mainly internal (which is why they're undocumented). For clients, however, it will only compile once and cache, and disabling features can lead to weird errors.